src/test/java/at/gammastrahlung/monopoly_server/game/GameBoardTests.java

```diff
@@ -97,4 +97,21 @@ void testInitializeChanceDeck() {
         assertEquals(CardType.GO_TO_JAIL, firstCard.getCardType());
         assertEquals(30, firstCard.getMoveToField());
     }
+
+
+    @Test
+    void testBoardNameInitialization() {
+        assertEquals("\uD83C\uDCCF️", gameBoard.getCommunityChestBoardName(), "Community Chest board name should have correct Unicode.");
```

Comment:
This has nothing to do with your feature.
Also this means that changing the display text requires changing the tests.